src/lib/updateGrantsRecipients.js

```diff
@@ -91,10 +92,11 @@ export async function processFiles() {
 
       const regionId = parseInt(g.region_id, 10);
       const cdi = regionId === 13;
+      // grant belonging to recipient's id 5 is merged under recipient's id 7782  (TTAHUB-705)
       return {
         id: parseInt(g.grant_award_id, 10),
         number: g.grant_number,
-        recipientId: parseInt(g.agency_id, 10),
+        recipientId: g.agency_id === '5' ? 7782 : parseInt(g.agency_id, 10),
```

Comment:
Should we put '5' in a constant so it is a little more clear why we are treating it special? Also maybe a link back to the ticket?